net-wireless/dump1090/dump1090-9999.ebuild

```diff
@@ -52,10 +45,10 @@ src_install() {
 	dodoc README.md
 
 	insinto /usr/share/${PN}/html
-	doins -r public_html/*
+	doins -r public_html
```

Comment:
you sure this gives the same result?

Reply:
Ah, actually no, this creates `/usr/share/${PN}/html/public_html` instead of just `/usr/share/${PN}/html/`. Fixed it :)